src/index.html

```diff
@@ -8,9 +8,42 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#" 
+      method="GET" 
+      data-qa="big" 
+    >
+      <div class="bigField">
+        <div class="text-Field_icon">
+          <input 
+            data-qa="keypress"
+            type="text" 
+            class="firstBar"
+            name="Big_Search_bar" 
+            placeholder="Try &quot Los Angeles &quot"
```

Comment:
```suggestion
            placeholder="Try &quot;Los Angeles&quot;"
```
Remove spaces after quotes